lib/constants.ts

```diff
@@ -0,0 +1,7 @@
+import { PrivilegeLevel } from '@prisma/client';
+
+const LOWEST_AUTHORIZED_PRIVILEGE_LEVEL = PrivilegeLevel.ADMIN;
```

Comment:
This should also be in privilegeLevelUtils. The primary question I have is should this be constant be used by any other file besides privilegeLevelUtils?

Reply:
Yea I agree- will move it